moduleroot/.github/workflows/cron.yml.erb

```diff
@@ -0,0 +1,94 @@
+name: Nightly tests
+
+on:
+  schedule:
+    - cron: '4 * * * *'
+
+jobs:
+  unit:
+    if: github.repository == '<%= @configs[:namespace] %>/<%= @configs[:puppet_module] %>'
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - "2.5"
+          - "2.4"
+        puppet:
+          - "6"
+          - "5"
+        exclude:
+          - ruby: "2.5"
+            puppet: "5"
+          - ruby: "2.4"
+            puppet: "6"
```

Comment:
should these be configured somewhere central and templated here?

Reply:
I thought about that, but we're fairly opinionated in our modules. Effectively what I want is to test Ruby 2.4 + Puppet 5 & Ruby 2.5 + Puppet 6 but AFAIK this is the only way to specify that.